.circleci/config.yml

```diff
@@ -2,24 +2,9 @@ version: 2
 jobs:
   build:
     docker:
-      - image: ubuntu:rolling
+      - image: blackjacktheviking/liblifthttpbuilder:0.1
```

Comment:
I think it would make sense to add a /docker like directory in the root and include the image build docker files in this project directly, what do you think?

Reply:
yeah agreed. I currently just threw them here while I waited to ask: https://github.com/clinekev11/liblifthttpbuilder